src/main/java/cz/xtf/TestConfiguration.java

```diff
@@ -125,494 +44,229 @@
 	public static final String VERSION_JDV = "xtf.version.jdv";
 	public static final String VERSION_JDG = "xtf.version.jdg";
 	public static final String VERSION_EWS = "xtf.version.ews";
-	public static final String VERSION_FUSE = "xtf.version.fuse";
+
 	public static final String VERSION_KIE = "xtf.version.kie";
 	public static final String VERSION_JDK = "xtf.version.jdk";
 	public static final String VERSION_SSO = "xtf.version.sso";
 	public static final String VERSION_AMQ = "xtf.version.amq";
 	public static final String VERSION_MSA = "xtf.version.msa";
 
-	public static final String CDK_DOMAIN = "cd.xtf.cz";
-	public static final String CDK_IP = "10.1.2.2";
-	public static final String CDK_INTERNAL_HOSTNAME = "localhost.localdomain";
+	public static final String VERSION_FUSE = "xtf.version.fuse";
 
-	public static final String OC_BINARY_LOCATION = "oc.binary.location";
+	public static final String CDK_INTERNAL_HOSTNAME = "localhost.localdomain";
 
 	public static final String CI_USERNAME = "ci.username";
 	public static final String CI_PASSWORD = "ci.password";
 
-	private static final Logger LOGGER = LoggerFactory.getLogger(TestConfiguration.class);
-	private static final TestConfiguration INSTANCE = new TestConfiguration();
-
-	private final Properties properties = new Properties();
-
-	private TestConfiguration() {
-		// first let's try product properties
-		copyValues(fromPath("test.properties"), true);
-
-		// then product properties
-		copyValues(fromPath("../test.properties"));
-
-		// then system variables
-		copyValues(System.getProperties());
 
-		// then environment variables
-		copyValues(fromEnvironment());
+	private Properties properties;
 
-		// Use global properties stored in repository - typically images - if not set before
-		if (Paths.get("global-test.properties").toAbsolutePath().toFile().exists()) {
-			copyValues(fromPath("global-test.properties"));
-		}
-		else if (Paths.get("../global-test.properties").toAbsolutePath().toFile().exists()) {
-			copyValues(fromPath("../global-test.properties"));
-		}
+	private static final TestConfiguration INSTANCE = new TestConfiguration();
 
-		// then defaults
-		copyValues(defaultValues());
+	protected TestConfiguration() {
```

Comment:
This class shouldn't be netiher instantiatable  at all nor it shouldn't hold any properties. Those are actually copied in `XTFConfiguration` class and there is no need to hold pointer also in this class. Constructor should be marked as private and this class properties instantiation should be moved to static block:

`static
{
get().copyValues(fromEnvironment());
get().copyValues(defaultValues());
}`

after that something like this should work without problem:
`get().readValue(TestConfiguration.IMAGE_MSA)`